src/workerd/api/worker-rpc.c++

```diff
@@ -148,11 +148,28 @@ private:
 
 } // namespace
 
-jsg::Promise<jsg::Value> JsRpcStub::sendJsRpc(
-    jsg::Lock& js,
-    rpc::JsRpcTarget::Client client,
-    kj::StringPtr name,
-    const v8::FunctionCallbackInfo<v8::Value>& args) {
+jsg::Promise<jsg::Value> JsRpcProperty::call(const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
I was reading through the code locally and realized I'm actually a bit confused how we get from `Fetcher::getRpcMethod()` to sending the rpc. 

When we return a `JsRpcProperty`, does the `JSG_CALLABLE(call)` in `JsRpcProperty` mean that in JS if I actually use `()` after the property name it will run this `JsRpcProperty::call()` method? This might be worth a comment somewhere.

Reply:
Yes, that's what JSG_CALLABLE means... the object can be called as a function.

Reply:
0bf67e4